rachelbuchta/refactor_tractor
# Feature/implement user

**What does this PR do?**
- adds implementation for the `User` class
- fleshes out a few more tests for `User`
**Is this a feature or a fix?**
- feature
**Where should the reviewer start?**
- `src/user.js`
**How should this be tested?**
- pull down branch (not to main)
- run `npm run test`
- observe all `User` tests passing
**This PR closes which open issues**
- Closes #26
